spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -222,4 +224,88 @@
       expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
     end
   end
+
+  context 'when remotePath is set in default stanza' do
+    let(:index_config) do
+      {
+        'config' => {
+          'default' => {
+            'remotePath' => 'volume:remote_store/$_index_name',
+            '_maxDailyDataSizeMB' => 100
+          },
+          'index_a' => {
+            '_maxDailyDataSizeMB' => 100
+          },
+          'index_b' => {
+            '_maxDailyDataSizeMB' => 25
+          }
+        },
+        'flags' => {
+          'index_a' => { 'noGeneratePaths' => true },
+          'index_b' => { 'noGeneratePaths' => true }
+        }
+      }
+    end
+
+    it 'writes the indexes.conf file with calculated maxGlobalDataSizeMB' do
+      expected_attributes = {
+        stanzas: {
+          'default' => {
+            'maxGlobalDataSizeMB' => 240_240,
+            'remotePath' => 'volume:remote_store/$_index_name'
+          },
+          'index_a' => {
+            'maxGlobalDataSizeMB' => 240_240
+          },
+          'index_b' => {
+            'maxGlobalDataSizeMB' => 60_060
+          }
+        }
+      }
+
+      expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
+    end
+  end
+
+  context 'when remotePath is set to true in specific stanza' do
```

Comment:
minor, remotePath isn't a boolean.  Reword this to something like 'when remotePath is set in specific stanza'.

Reply:
updated [43ed5fe](https://github.com/cerner/cerner_splunk/pull/241/commits/43ed5fe18075cff99d99acbdcc302e2427146dbb#diff-2dbf75fe595324af1923d6c51037b9e78cab1adcc9d842af08ba7e0efba026d4R314)